### ISSUE TITLE
Inherit documentation for procedures implementing interface methods

### ISSUE THREAD

**Issue by rvanbekkum** [365businessdev/vscode-alxmldocumentation#27]
Dear @365businessdev team,
Would it be possible to have the AL-XML-DOC code-analyzer take into account procedures implementing an `interface` method?
Considering you will mostly invoke the procedure through an interface variable, requiring documentation for procedures implementing it, might not always be that useful. Of course you add some additional comments for the specific implementation, but the purpose of the procedure and its parameters will be the same and most likely won't need documentation again.
Maybe the extension could do something with `<inheritdoc/>` to account for this?

**Comment by sirhc101** [comment]
I like this approach very much, because I have been thinking about this "problem" for a while and I am looking for a solution. I will discuss this with the team and get back to you.

**Comment by rvanbekkum** [comment]
Hi @365businessdev,
I noticed that you marked this feature as released, but unfortunately it does not seem to work.
I have a codeunit that implements an interface, the interface has documentation for its procedures, but I still get the message that documentation is missing, also after adding `<inheritdoc/>`.
How was this implemented, or is this missing or a bug?

**Comment by sirhc101** [comment]
Hi @rvanbekkum 

Where did you get the missing documentation message? Do we talk about VSCode directly or the Export to Markdown?
What we've implemented in the current version is this:
![GitHub#27](https://user-images.githubusercontent.com/39621236/100843123-2cf76f80-347a-11eb-948b-2a0c3047919a.gif)

The Export to Markdown does currently not support `<inheritdoc/>`, but we are releasing a new version shortly.

**Comment by sirhc101** [comment]
Let me add one notice:
Why did we not simply implemented `<inheritdoc/>` and added the code reference (`cref`)?
The reason for this are overloaded procedures. When we just used `<inheritdoc/>` we had problems to identify the exact procedure in Interface object.
That's why we decided to use `<inheritdoc cref="Reference"/>` instead.

**Comment by rvanbekkum** [comment]
Hi @sirhc101,
It seems like I am not getting these options from the IntelliSense. I looked into it, and it seems like this happens when the codeunit object name contains an underscore `_`. Could you have a look at this?

**Comment by sirhc101** [comment]
Hi @rvanbekkum,

Do you not see the `AL XML Documentation Interface Comment` and `Inherit AL XML Documentation Comment` completion (IntelliSense) items after typing `///` or does the code analyzer compain about missing documentations?

In case you do not see the completion items it looks like the extension cannot access the Interface object source code (`showMyCode`?) 

I tried with an new Interface object and a new Codeunit object with underscore, but for me it looks ok!?
```js
interface "My_Interface"
{
    /// <summary>
    /// MyProcedure.
    /// </summary>
    procedure MyProcedure();

    /// <summary>
    /// MyAdvancedProcedure.
    /// </summary>
    procedure MyAdvancedProcedure();
}

codeunit 50113 "My Interface Impl." implements My_Interface
{
    /// <inheritdoc cref="My_Interface.MyProcedure()"/>
    procedure MyProcedure();
    begin

    end;

    /// <inheritdoc cref="My_Interface.MyAdvancedProcedure()"/>
    procedure MyAdvancedProcedure();
    begin

    end;
}
```

**Comment by rvanbekkum** [comment]
Hi @sirhc101,
I am not seeing the completion items "when the **codeunit** object name contains an underscore `_`" 😉.
This also happens if the interface and codeunit are in the same project.

**Comment by rvanbekkum** [comment]
More specifically, it's for a codeunit object name without surrounding quotes `"` and with an underscore in the name `_`.

**Comment by sirhc101** [comment]
> I am not seeing the completion items "when the **codeunit** object name contains an underscore `_`" 😉.

Oops! It's early 😉 
No I can reproduce. I will investigate and come back to you.

Thank you.

**Comment by sirhc101** [comment]
I've found the issue and fixed it in dev-branch. Let me test a little more with other combinations.

**Comment by sirhc101** [comment]
Sorry for the delay in fixing your supported issue!
We have recently released the version 1.0.5 which fixed this issue.
